packages/rfw/lib/src/flutter/core_widgets.dart

```diff
@@ -644,7 +644,7 @@ Map<String, LocalWidgetBuilder> get _coreWidgetsDefinitions => <String, LocalWid
       locale: ArgumentDecoders.locale(source, ['locale']),
       softWrap: source.v<bool>(['softWrap']),
       overflow: ArgumentDecoders.enumValue<TextOverflow>(TextOverflow.values, source, ['overflow']),
-      textScaleFactor: source.v<double>(['textScaleFactor']),
+      textScaleFactor: source.v<double>(['textScaleFactor']),  // ignore: deprecated_member_use
```

Comment:
I wonder how this is going to be migrated, since `textScaleFactor` is essentially a function?

Reply:
would something like `textScaler: TextScaler.linear(source.v<double>(['textScaleFactor']))` work?